boards/qemu_i486_q35/Makefile

```diff
@@ -16,12 +16,15 @@ WORKING_QEMU_VERSION := 7.2.0
 # Peripherals attached by default:
 # - 16550 UART (attached to stdio by default)
 # - VGA display (pass NOVGA=1 to disable)
+# - Virtio EntropySource RNG (attached to /dev/urandom)
 QEMU_BASE_CMDLINE := \
   $(QEMU_CMD) \
     -cpu 486 \
     -machine q35 \
     -net none \
-    -device isa-debug-exit,iobase=0xf4,iosize=0x04
+    -device isa-debug-exit,iobase=0xf4,iosize=0x04 \
+    -object rng-random,filename=/dev/urandom,id=rng0 \
+    -device virtio-rng-pci,rng=rng0,disable-legacy=on
```

Comment:
Can you add these extra arguments to `.cargo/config.toml` so that it can be run with `cargo run`?

Reply:
Done in most recent push